### PR TITLE
Add Binjie Yang as committer

### PR DESCRIPTION
**Pull request by zwangsheng**
As title.

### CODE REVIEW

**Comment by zwangsheng** [comment]
cc @pan3793 @yaooqinn

File: src/en/news/2023-10-11-new-committers.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: '新的 committer: Binjie Yang'
```
Comment (pan3793) [inline src/en/news/2023-10-11-new-committers.md:2]: it's en

File: src/en/news/2023-10-11-new-committers.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+The Project Management Committee (PMC) for Apache Kyuubi
+has invited [Binjie Yang](https://github.com/zwangsheng) to become a committer and we are pleased
+to announce that he has accepted.
+
+Binjie Yang works at NetEase Games, he
+
+- started contribution to Kyuubi community since Mar. 2021
+- made a major contribution to fit Kyuubi with Spark Engine On Kubernetes
+- participate in Kyuubi On Kubernetes adaptation and integration test construction
+- join the construction of Kyuubi Web UI & Restful Api V2
```
Comment (pan3793) [inline src/en/news/2023-10-11-new-committers.md:30]:
```suggestion
- join the construction of Kyuubi Web UI & Batch V2
```

File: src/zh/become_committer.md [hunk 3]
```diff
@@ -91,6 +96,7 @@ menu:
       <td>yao</td>
       <td><a href="https://github.com/yaooqinn">yaooqinn</a></td>
     </tr>
+    <tr>
     <td>Paul Lin</td>
```
Comment (pan3793) [inline src/zh/become_committer.md:100]: nit: indent

File: src/zh/news/2023-10-11-new-committers.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Apache Kyuubi 的项目管理委员会已经邀请[Binjie Yang](https://github.com/zwangsheng)
+成为我们的 Committer，我们很高兴地宣布他已经接受邀请。
+
+Binjie Yang 现就职于网易, 他
+
+- 自 2021 年 3 月以来，参与 Kyuubi 社区贡献
+- Kyuubi Spark Engine On Kubernetes 适配的主要贡献者
+- 参与 Kyuubi On Kubernetes 适配以及集成测试构建
+- 参与 Kyuubi Web UI & Restful Api V2 建设工作
```
Comment (pan3793) [inline src/zh/news/2023-10-11-new-committers.md:29]:
```suggestion
- 参与 Kyuubi Web UI & Batch V2 建设工作
```

File: src/en/news/2023-10-11-new-committers.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+The Project Management Committee (PMC) for Apache Kyuubi
+has invited [Binjie Yang](https://github.com/zwangsheng) to become a committer and we are pleased
+to announce that he has accepted.
+
+Binjie Yang works at NetEase Games, he
```
Comment (ulysses-you) [inline src/en/news/2023-10-11-new-committers.md:25]: please do not copy without head...

Comment (zwangsheng) [reply src/en/news/2023-10-11-new-committers.md]: Addressed, XD

**Review by yaooqinn — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Comment by yaooqinn** [comment]
Congrats!

**Review by bowenliang123 — APPROVED** [review]

**Review by cfmcgrady — APPROVED** [review]
Congrats!

**Review by pan3793 — APPROVED** [review]
Congrats!

**Comment by zwangsheng** [comment]
Thanks all, merged.

**Pull request merged**